src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASection.tsx

```diff
@@ -1,149 +1,134 @@
 import { ContextModule, OwnerType } from "@artsy/cohesion"
-import { Button, Clickable, Separator, Text } from "@artsy/palette"
-import { useTracking } from "react-tracking"
+import { Box, Button, Clickable, Text } from "@artsy/palette"
+import { createOrUpdateConsignSubmission } from "Apps/Consign/Routes/SubmissionFlow/Utils/createOrUpdateConsignSubmission"
 import { RouterLink } from "System/Components/RouterLink"
+import { useFeatureFlag } from "System/Hooks/useFeatureFlag"
+import { useRouter } from "System/Hooks/useRouter"
+import { useSystemContext } from "System/Hooks/useSystemContext"
+import { getENV } from "Utils/getENV"
+import { MyCollectionArtworkSWASection_artwork$key } from "__generated__/MyCollectionArtworkSWASection_artwork.graphql"
+import { useState } from "react"
+import { graphql, useFragment } from "react-relay"
+import { useTracking } from "react-tracking"
 
-interface Props {
-  route: string
+interface MyCollectionArtworkSWASectionProps {
   learnMore: () => void
-  slug: string
-  artworkId: string
+  artwork: MyCollectionArtworkSWASection_artwork$key
   ctaColor?: "primaryBlack" | "secondaryNeutral" | null
 }
 
-export const MyCollectionArtworkSWASectionMobileLayout: React.FC<Props> = ({
-  route,
+export const MyCollectionArtworkSWASection: React.FC<MyCollectionArtworkSWASectionProps> = ({
```

Comment:
No need to have separate components for desktop and mWeb. I've merged both to simplify this component.